lib/ecto/migration.ex

```diff
@@ -768,9 +772,15 @@ defmodule Ecto.Migration do
 
       execute "CREATE EXTENSION postgres_fdw", "DROP EXTENSION postgres_fdw"
 
+      alias Ecto.Adapters.SQL
+
+      execute(&execute_up/0, &execute_down/0)
+
+      defp execute_up, do: SQL.query!(repo(), "select 'Up query …';", [], [log: :info])
+      defp execute_down, do: SQL.query!(repo(), "select 'Down query …';", [], [log: :info])
```

Comment:
`repo().query!`should also work. (i.e. no need to alias SQL).

Reply:
@josevalim Sorry, did I missed something? I do not see `query!` callback …

https://hexdocs.pm/ecto/Ecto.Repo.html#callbacks